apps/taxes/src/modules/taxjar/taxjar-calculate-taxes-adapter.ts

```diff
@@ -0,0 +1,141 @@
+import { TaxForOrderRes } from "taxjar/dist/types/returnTypes";
+import { TaxBaseFragment } from "../../../generated/graphql";
+import { ChannelConfig } from "../channels-configuration/channels-config";
+import { distributeDiscount } from "../taxes/distribute-discount";
+import { numbers } from "../taxes/numbers";
+import { taxProviderUtils } from "../taxes/tax-provider-utils";
+import { CalculateTaxesResponse } from "../taxes/tax-provider-webhook";
+import { taxJarAddressFactory } from "./address-factory";
+import { FetchTaxForOrderArgs, TaxJarClient } from "./taxjar-client";
+import { TaxJarConfig } from "./taxjar-config";
+import { WebhookAdapter } from "../taxes/tax-webhook-adapter";
+
+type Payload = {
+  taxBase: TaxBaseFragment;
+  channel: ChannelConfig;
+};
+
+type Target = FetchTaxForOrderArgs;
+type Response = CalculateTaxesResponse;
+
+export class TaxJarCalculateTaxesPayloadTransformer {
```

Comment:
I think we should stick here to 1 class per file, especially if you export them anyway